### PR TITLE
AdminUser reactivate action

### PR DESCRIPTION
**Pull request by lerrua**

### CODE REVIEW

File: src/c/admin-contribution-user.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+window.c.AdminContributionUser = (function(m) {
```
Comment (diogob) [inline src/c/admin-contribution-user.js]: Let's add a doctoring explaining the component here.
For an example you can take a look here: https://github.com/catarse/catarse.js/blob/a0ac0c21ba6820456628b81e46d3f06a1e55e951/src/c/admin-external-action.js#L1

File: src/c/admin-user-detail.js [hunk 2]
```diff
@@ -1,13 +1,43 @@
-window.c.AdminUserDetail = (function(m, _, c) {
+window.c.AdminUserDetail = (function(m, _, c){
```
Comment (diogob) [inline src/c/admin-user-detail.js]: Same here about the doctoring

File: src/c/admin-input-action.js [hunk 3]
```diff
@@ -69,11 +70,11 @@ window.c.AdminInputAction = (function(m, h, c) {
                         m('input.w-button.btn.btn-small[type="submit"][value="' + btnValue + '"]')
                     ] : (!ctrl.error()) ? [
                         m('.w-form-done[style="display:block;"]', [
-                            m('p', 'Apoio transferido com sucesso!')
+                            m('p', data.successMessage)
```
Comment (diogob) [inline src/c/admin-input-action.js]: :+1:

File: src/c/admin-contribution-user.js (hunk 1 above)
Comment (lerrua) [inline src/c/admin-contribution-user.js]: done :kiss:

File: src/c/admin-contribution-user.js [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+        view: (ctrl, args) => {
+            const item = args.item,
+                  user = {
+                      profile_img_thumbnail: item.user_profile_img,
+                      id: item.user_id,
+                      name: item.user_name,
+                      email: item.email,
+                  };
+
+            const additional_data = m('.fontsize-smallest.fontcolor-secondary', 'Gateway: ' + item.payer_email);
```
Comment (vicnicius) [inline src/c/admin-contribution-user.js]: I think we are only using camel case to name our vars on catarse.js.

Comment (lerrua) [inline src/c/admin-contribution-user.js]: good point

**Pull request merged**